coderd/database/db.go

```diff
@@ -18,6 +18,13 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// Well-known lock IDs for lock functions in the database. These should not
+// change. If locks are deprecated, they should be kept to avoid reusing the
+// same ID.
+const (
+	LockIDDeploymentSetup = iota + 1
```

Comment:
Why `iota + 1`?

Reply:
This is just how it was... seemed risky to change as part of this.

Reply:
I don't know what the original reason was, but a typical reason is to avoid mixing it with the zero value. FWIW.